src/utils/daos_hdlr.c

```diff
@@ -3701,101 +3407,88 @@ cont_clone_hdlr(struct cmd_args_s *ap)
 	}
 
 	rc = dm_connect(ap, is_posix_copy, &dst_cp_type, &src_cp_type,
-			&ca, ap->sysname, ap->dst, &src_cont_info,
-			&dst_cont_info);
+			&ca, ap->sysname, ap->dst, &src_cont_info, &dst_cont_info);
 	if (rc != 0) {
 		D_GOTO(out_disconnect, rc);
 	}
-	rc = daos_cont_create_snap_opt(ca.src_coh,
-				       &epoch, NULL,
-			       DAOS_SNAP_OPT_CR | DAOS_SNAP_OPT_OIT, NULL);
+	rc = daos_cont_create_snap_opt(ca.src_coh, &epoch, NULL,
+       				       DAOS_SNAP_OPT_CR | DAOS_SNAP_OPT_OIT, NULL);
```

Comment:
(style)  code indent should use tabs where possible
(style)  please, no space before tabs